src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/config.jelly

```diff
@@ -37,6 +37,17 @@
     <f:entry title="${%API URL Override}" field="apiUrlOverride">
       <f:textbox/>
     </f:entry>
+      <f:section title="BrowserSelection">
+          <f:entry title="Browser" field="browser">
+              <f:checkbox name="chrome" title="Chrome" checked="${browsers.contains('chrome')}" />
```

Comment:
We were able to add better padding around these checkboxes and labels, so they are not directly touching?